src/pages/build-apps/guides/transaction-signing.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Transaction signing
+description: Prompt users to sign and broadcast transactions to the Stacks blockchain
+experience: beginners
+tags:
+  - tutorial
+images:
+  large: /images/transaction-signing.svg
+  sm: /images/transaction-signing.svg
+---
+
+## Introduction
+
+This guide explains how to prompt users to sign [transactions](/understand-stacks/transactions) and broadcast them to the Stacks blockchain by implementing the `connect` package of [Stacks.js](https://blockstack.github.io/stacks.js/).
```

Comment:
maybe link to connect instead of stacks.js - I think we do this in the other guides?